course/forms.py

```diff
@@ -32,6 +32,7 @@ def clean(self):
         self.member_profiles = [self.profile]
         for key in self.fields.keys():
             if key in self.cleaned_data and self.cleaned_data[key]:
+                # PS: I guess this is not covered by issue #600
```

Comment:
I suppose this comment can be removed from the final version.

Reply:
yes